src/register/RegistrationPage.jsx

```diff
@@ -673,16 +651,10 @@ class RegistrationPage extends React.Component {
             <StatefulButton
               type="submit"
               variant="brand"
-              className={classNames(
-                'mt-4 mb-4',
-                { 'stateful-button-variation1-width': this.state.registerRenameExpVariation === 'variation1' },
-                { 'stateful-button-width': this.state.registerRenameExpVariation !== 'variation1' },
```

Comment:
Can we remove `stateful-button-width` from scss file as well?

Reply:
It a common style; it is being used for reset page form button